custom/tests.yaml

```diff
@@ -3284,17 +3311,28 @@ api:
         return false;
       }
 
+      // The following function is defined to help obtain individual stats types
+      // and entries
+      function getStatByTypeAndEntry(type, entry) {
+        for (var i = 0; i < stats.length; i++) {
+          if (stats[i].type === type
+            && Object.prototype.hasOwnProperty.call(stats[i], entry)
+            && stats[i][entry] !== null) {
+            return stats[i];
+          }
+        }
+        return false;
+      }
+
       var instance = reusableInstances.rtcStats;
     __additional:
       type_candidate-pair: |-
         var stat = getStatByType('candidate-pair');
         return !!stat;
       type_candidate-pair.availableIncomingBitrate: |-
-        var stat = getStatByType('candidate-pair');
-        return !!stat && 'availableIncomingBitrate' in stat;
+        return !!getStatByTypeAndEntry('candidate-pair', 'availableIncomingBitrate');
       type_candidate-pair.availableOutgoingBitrate: |-
-        var stat = getStatByType('candidate-pair');
-        return !!stat && 'availableOutgoingBitrate' in stat;
+        return !!getStatByTypeAndEntry('candidate-pair', 'availableOutgoingBitrate');
       type_candidate-pair.bytesDiscardedOnSend: |-
         var stat = getStatByType('candidate-pair');
         return !!stat && 'bytesDiscardedOnSend' in stat;
```

Comment:
For consistency, we should probably use the `getStatByTypeAndEntry()` function everywhere (or the similar behavior on `getStatByType(type, entry)` if you'd like to go that route)!